packages/core/useScroll/index.ts

```diff
@@ -251,7 +251,7 @@ export function useScroll(
   )
 
   tryOnMounted(() => {
-    setArrivedState(toValue(element))
+    toValue(element) && setArrivedState(toValue(element))
```

Comment:
I was literally opening a PR with this change but before doing that I checked and I saw this PR.

I think the below suggestion follows the guidelines better, given similar code logic in this file follows this style 🙇 

```suggestion
    const _element = toValue(element)
    if (!_element)
      return

    setArrivedState(_element)
```